File: InventoryService/InventoryService/HDModels/DataRecords.cs
using System;
using System.Collections.Generic;

namespace InventoryService.HDModels
{
    public partial class DataRecords
    {
        public int RecordId { get; set; }
        public string Mac { get; set; }
        public string Category { get; set; }
        public int Data { get; set; }
        public string Position { get; set; }
        public DateTime CreationTime { get; set; }
        public DateTime SendTime { get; set; }
        public DateTime DbTime { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/example/otglister/DataRecords.kt
package com.example.otglister

import android.os.Parcel
import android.os.Parcelable
import java.sql.Time
import java.io.Serializable

public data class DataRecords(
    val Mac: String?,
    val Category: String?,
    val Data: Int,
    val CreationTime: String?,
    val SendTime: String? )  {

}